### DISCUSSION TITLE
Help With Integration

### DISCUSSION THREAD

**Discussion by AudiHazael** [emailjs-com/help#24] [Q&A]
I am new into website development, and this form integration with EmailJS is a pain in the ass. I already have a form form my website, and I also have a script that makes the form interactive and all that. Integrating seems not to be working. Is there anyone that can help take a look at my code, and show me a way out? I've tried following all the steps, but it seems like I am missing something.

**Comment by AyrtonBrooke** [comment]
i can help if you are using gmail

**Comment by xr0master** [comment]
I'm sure the community will be happy to help you, but you have to explain what doesn't work for you.
